PC/layout/main.py

```diff
@@ -66,6 +66,17 @@
 TOOLS_DIRS = FileNameSet("scripts", "i18n", "pynche", "demo", "parser")
 TOOLS_FILES = FileSuffixSet(".py", ".pyw", ".txt")
 
+def copy_if_modified(s, d):
+    dest = Path(d)
+    source = Path(s)
+
+    if dest.is_file():
+        timeNotEqual = True if s.stat().st_mtime !=dest.stat().st_mtime else False
+        sizeNotEqual = True if s.stat().st_size != dest.stat().st_size else False
+
+    if not dest.is_file() or timeNotEqual or sizeNotEqual:
+        print(dest)
+        shutil.copy2(str(s), str(dest))
```

Comment:
(Untested, but was easier to rewrite than point out all the things to fix ;) )

```python
def copy_if_modified(src, dest):
    try:
        dest_stat = os.stat(dest)
    except FileNotFoundError:
        do_copy = True
    else:
        src_stat = os.stat(src)
        do_copy = (src_stat.st_mtime != dest_stat.st_mtime or
                   src_stat.st_size != dest_stat.st_size)

    if do_copy:
        shutil.copy2(src, dest)
```

Reply:
Thanks.  I'll test it.